frontend/ui/elements/reader_menu_order.lua

```diff
@@ -24,6 +24,7 @@ local order = {
     typeset = {
         "set_render_style",
         "style_tweaks",
+        "font_settings",
         "----------------------------",
         "change_font",
```

Comment:
Better be moved below the separator (that separates style stuff from font stuff)

Reply:
And moved again below `change_font` ? :)
(Because I'm so used to find it on the 3rd position, and easthetically, having the following longer items together should look better, instead of having the little "Font >" in between.
(Would it need an added `separator=true` for that `Font settings>`, that would appear only on isDesktop?)

Reply:
I'd be inclined to move it down one further in line with the general menu order, which would add an extra separator, Though in this case it's defensible that change font, hyphenation & hanging punctuation belong closer together.

Reply:
**non Desktop vs Desktop**
![Sin nombre](https://user-images.githubusercontent.com/975883/63357445-bbb9a900-c369-11e9-9411-2876d7bd1dce.png)

¿?

Reply:
It's fine I guess, but like I said initially, I think it'd be best within the font menu itself.

Reply:
I'm fine with that screenshot.

Reply:
> It's fine I guess, but like I said initially, I think it'd be best within the font menu itself.

Ok, that makes sense. I didn't understand.

Reply:
Needs to stay on top of that menu because where system fonts are enabled the list of pages can grow a bit:

![Sin nombre](https://user-images.githubusercontent.com/975883/63360216-43091b80-c36e-11e9-9097-cb8ed7d4be22.png)